pkgs/development/python-modules/ijson/default.nix

```
@@ -39,5 +45,6 @@ buildPythonPackage rec {
     homepage = "https://github.com/ICRAR/ijson";
     changelog = "https://github.com/ICRAR/ijson/blob/v${version}/CHANGELOG.md";
     license = licenses.bsd3;
+    maintainers = with maintainers; [ ];
```

Can I be a maintainer? :3

```suggestion
    maintainers = with maintainers;  [ AndersonTorres ];
```

---

You can add yourself to all packages to want to maintain.